tests/ml_commons/test_model_access_control.py

```diff
@@ -0,0 +1,250 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+import os
+import time
+from unittest.mock import patch
+
+import pytest
+from opensearchpy.exceptions import NotFoundError, RequestError
+from packaging.version import parse as parse_version
+
+from opensearch_py_ml.ml_commons.model_access_control import ModelAccessControl
+from tests import OPENSEARCH_TEST_CLIENT
+
+OPENSEARCH_VERSION = parse_version(os.environ.get("OPENSEARCH_VERSION", "2.7.0"))
```

Comment:
is 2.7.0 is the default value here?

Reply:
Yes. Default version will be 2.7.0. But, this is only for the local cases. Since, on github actions, we always set a value to `OPENSEARCH_VERSION`. We can still try to get this from the test client by actually doing an API call.

Reply:
By local you mean the development environment, right? Can we make 2.11 default?

Reply:
Sure. I'll update that.